app/assets/javascripts/discourse/widgets/topic-admin-menu.js.es6

```diff
@@ -52,7 +52,10 @@ createWidget("topic-admin-menu-button", {
         this.attach("button", {
           className:
             "btn-default toggle-admin-menu" +
-            (attrs.fixed ? " show-topic-admin" : ""),
+            (attrs.fixed ? " show-topic-admin" : "") +
+            (attrs.triggerKeyboardShortcut
```

Comment:
This is a bit of a nit pick, but could we call this `addKeyboardTargetClass` and then call it `keyboard-target-admin-menu` or something similar? I think it's more clear that the attribute means this is a target for keyboard operations rather than a trigger for them.

Reply:
Sure, I completely agree with you.